homeassistant/components/panasonic_viera/__init__.py

```diff
@@ -72,6 +92,14 @@ async def async_setup_entry(hass, config_entry):
     host = config[CONF_HOST]
     port = config[CONF_PORT]
 
+    if CONF_LISTEN_PORT not in config:
+        config_entry = hass.config_entries.async_update_entry(
```

Comment:
Is it ok to update older config entries or should I just expect people to reconfigure?

Reply:
It's good to update like this.